src/Execute/DeviceInstance.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DeviceSpec} from '../Backend/Spec';
+
+// What is `RealDeviceInstance` class?
+// Suppose a case that some tizen tvs are. dtv0, dtv1, ... dtvn_1
+// the dtv's spec can be just only one
+// However, each dtv should be handled one by one
+// At the time, this class expresses an instance for a real device
```

Comment:
```suggestion
// Suppose the case of multiple tizen tvs - dtv0, dtv1, ... dtvn_1
// Every dtv instances shares one device spec, but should be handled separately
// This class expresses an instance for each of such real devices
```

(Just suggestion) When I read the comment, it took time to understand what this `DeviceInstance` implemented for. I'm not good at writing down the explanation, but could you please consider about update the comment a little bit? (I'm not confident to the suggested version too.)

Reply:
I will revise this comment with change class name `Device`. I think class name could make more people hard to understand this concept, as this contains duplicated meaning.